src/indexPage/react-components/bot-landing/SwitchSection.jsx

```diff
@@ -2,6 +2,18 @@ import React from 'react';
 import { translate } from '../../../common/i18n';
 import { getOAuthURLDeriv } from '../../../common/appId';
 
+const getnerateURL = (url) => {
```

Comment:
Can we put this method somewhere else in a shared folder like utility ? so no need to repeat it twice